src/display/display_race_report.py

```diff
@@ -0,0 +1,41 @@
+from datetime import timedelta
+from enum import StrEnum
+
+from formula1_race_analysis import Driver, format_lap_time
+
+THE_NUMBER_OF_FASTEST_DRIVERS_PASSED_Q1 = 15
+
+
+class SortStrategy(StrEnum):
+    ASCENDING_ORDER = "asc"
+    DESCENDING_ORDER = "desc"
+
+
+def display_race_report(report: list[tuple[Driver, timedelta]]) -> None:
+    """
+    Displays a formatted race report.
+    """
+    for position, data in enumerate(report, start=1):
+        driver, lap_time = data
+        formatted = format_lap_time(lap_time)
+        print(f"{position}. {driver.name} | {driver.car_model} | {formatted}")
+        if position == THE_NUMBER_OF_FASTEST_DRIVERS_PASSED_Q1:
+            print("_" * 60)
```

Comment:
![image](https://github.com/user-attachments/assets/ec4e7183-07b2-426f-8c06-cfbfb52fcb1c)
Can I requrest some line alignment?

driver 1   | car            |  time
driver 2   | bigger car |  time

So it has | at the same location